src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -364,7 +377,7 @@ class BasePaymentsPage extends React.Component {
                         isDangerousAction
                     />
                     <ConfirmPopover
-                        contentStyles={[!this.props.isSmallScreenWidth ? styles.sidebarPopover : '']}
+                        contentStyles={!this.props.isSmallScreenWidth ? [styles.sidebarPopover] : undefined}
```

Comment:
Why change to `undefined` instead of `[""]` which was there previously? Just curious.

Otherwise just missing the comments and we should be good to go.

Reply:
Setting it to undefined is like resetting it. So it will automatically take dfaulrProp value for this prop.

Reply:
Gotcha! thanks for the explanation.